docs/photoniq/functions/configuring-functions.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 4
+title: Configuring Functions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Depending on the language/framework used, all PhotonIQ functions come with a `photoniq.toml` file when they are created. The `photoniq.toml` file allows you to customize the behavior of your functions by configuring settings such as CORS, logging,  metadata, resource limits, and environment variables.
+
+The following table provides a detailed explanation of the fields available in the `photoniq.toml` file for configuring Rust, JS and NextJS functions:
+
+
+
+<Tabs groupId="languages">
+
+<TabItem value="rust/js" label="Rust/Javascript">
+
+```toml title='photoniq.toml'
+name = "{{function_name}}"
+version = "0.0.1"
+description = "Description for function {{function_name}}"
+lang = "rust"
+#execute_url_suffix = "optional_url_suffix_to_execute_function"
+
+[cors_settings]
+allowed_methods = ["GET","POST"]
+allowed_hosts = ["petstore.swagger.io"]
+allow_http = true
+
+[log_settings]
+enabled = true
+level = "INFO"
+
+[env_vars]
+MESSAGE = "Hello 👋! This message comes from an environment variable"
+
+```
+
+The table below describes the configuration options in the the `photoniq.toml`:
+
+| Field                          | Description                                                                |
+|--------------------------------|----------------------------------------------------------------------------|
+| `name`                         | The name of the function.                                                  |
+| `version`                      | The version of the function.                                               |
+| `description`                  | A brief description of the function.                                       |
+| `lang`                         | The programming language used for the function, in this case, `rust` or `js`.     |
+| `execute_url_suffix`           | Customizes the [function execution endpoint](https://www.macrometa.com/docs/apiFaas#/operations/handle_latest_mm_execute). For example, setting `execute_url_suffix = "my/prefix"` allows the function to be executed via `/my/prefix/myFunction` instead of the default `/api/faas/v1/execute/myFunction`.                      |
```

Comment:
```suggestion
| `execute_url_suffix`           | Customizes the [function execution endpoint](https://www.macrometa.com/docs/apiFaas#/operations/handle_latest_mm_execute). For example, setting `execute_url_suffix = "/my/path/suffix"` allows the function to be executed via `/my/path/suffix/myFunction` instead of the default `/api/faas/v1/execute/myFunction`, like an alias.                      |
```